build.gradle

```diff
@@ -27,7 +27,7 @@ buildscript {
   }
   dependencies {
     classpath 'gradle.plugin.com.github.jengelman.gradle.plugins:shadow:7.0.0'
-    classpath 'com.palantir.baseline:gradle-baseline-java:4.0.0'
+    classpath 'com.palantir.baseline:gradle-baseline-java:4.42.0'
```

Comment:
Nit / non-blocking: Should we separate the error prone changes from the upgrade itself? Not a big deal to me.